Lib/graphlib.py

```diff
@@ -132,14 +135,14 @@ def get_ready(self):
         return result
 
     def is_active(self):
-        """Return ``True`` if more progress can be made and ``False`` otherwise.
+        """Return True if more progress can be made and False otherwise.
 
-        Progress can be made if cycles do not block the resolution and either there
-        are still nodes ready that haven't yet been returned by "get_ready" or the
-        number of nodes marked "done" is less than the number that have been returned
-        by "get_ready".
+        Progress can be made if cycles do not block the resolution and either
+        there are still nodes ready to be returned by "get_ready()" or there
```

Comment:
Developers spend a lot of time looking at history, to find who changed some line when and for what reason.
Preserving the usefulness of that history is more important that strict line lengths, so here for example I would not rewrap the lines.